docs/pages/_document.js

```diff
@@ -106,6 +106,15 @@ export default class MyDocument extends Document {
               __html: `@font-face{font-family:'PlusJakartaSans-Bold';font-style:normal;font-weight:700;font-display:swap;src:url('/static/fonts/PlusJakartaSans-Bold-subset.woff2') format('woff2');}`,
             }}
           />
+          <style
+            // Loads General Sans: Regular (400), Medium (500), SemiBold (600), Bold (700)
+            // Typeface documentation: https://www.fontshare.com/fonts/general-sans
+            // use https://cssminifier.com/ to minify
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={{
+              __html: `@font-face{font-family:'General Sans';src:url(/static/fonts/GeneralSans-Regular.woff2) format('woff2'),url(/static/fonts/GeneralSans-Regular.ttf) format('truetype');font-weight:400;font-style:normal;font-display:swap;}@font-face{font-family:'General Sans';src:url(/static/fonts/GeneralSans-Medium.woff2) format('woff2'),url(/static/fonts/GeneralSans-Medium.ttf) format('truetype');font-weight:500;font-style:normal;font-display:swap;}@font-face{font-family:'General Sans';src:url(/static/fonts/GeneralSans-SemiBold.woff2) format('woff2'),url(/static/fonts/GeneralSans-SemiBold.ttf) format('truetype');font-weight:600;font-style:normal;font-display:swap;}@font-face{font-family:'General Sans';src:url(/static/fonts/GeneralSans-Bold.woff2) format('woff2'),url(/static/fonts/GeneralSans-Bold.ttf) format('truetype');font-weight:700;font-style:normal;font-display:swap;}`,
```

Comment:
@danilo-leal The font changes on hard reload. 🤔

https://github.com/mui/material-ui/assets/18292247/bdb8edd2-35f8-4700-a012-d1fd59037804

Reply:
Any clues about why that's happening? 😞

Reply:
Interesting to realize that this happens in the current production website, too, not with Jakarta though but rather, with IBM 🤔 

https://github.com/mui/material-ui/assets/67129314/ae6b879c-ad28-48f6-ba37-2009c7320197

Reply:
1. Are you using `GeneralSans` everywhere or just the tagline on marketing pages?
2. Do you plan to use PlusJarkarta elsewhere? If no, we should remove it.

Reply:
1. Using on the docs, too (e.g. `MarkdownElement`)
2. We can ditch it!

Let me know if you need anything else (replying to the comment below)!